packages/mds-agency-cache/logger.ts

```diff
@@ -0,0 +1,3 @@
+import logger from '@mds-core/mds-logger'
+
+export const AgencyCacheLogger = logger.createLogger('AgencyCache')
```

Comment:
Should we use `mds-agency-cache` as the name to make associating the log with the package easier?

Reply:
I thought about copy-pastaing the package name, but it feels a little redundant considering everything here has the `mds` prefix already added inherently by `mds-logger`

Reply:
I don't think it's redundant. `DEBUG=mds` is not the same as `DEBUG=mds-agency-cache` and if we use the package name by convention then I can turn on debug logs without having to refer to the code to find out what namespace is being used.